src/vgg.jl

```diff
@@ -0,0 +1,90 @@
+using Flux: convfilter, Zeros, outdims
+
+# Build a VGG block
+#  ifilters: number of input filters
+#  ofilters: number of output filters
+#  batchnorm: add batchnorm
+function vgg_block(ifilters, ofilters, depth, batchnorm)
+  k = (3,3)
+  p = (1,1)
+  layers = []
+  for l in 1:depth
+    if batchnorm
+      w = convfilter(k, ifilters=>ofilters)
+      b = Zeros()
+      push!(layers, Conv(weight=w, bias=b, pad=p))
+      push!(layers, BatchNorm(ofilters, relu))
```

Comment:
Same comment

Reply:
Just a quick question, here Conv layer takes directly weight and bias and b = Flux.Zeros(). According to [Flux.jl's docs](https://fluxml.ai/Flux.jl/stable/models/layers/#Flux.Conv) "Setting bias to Flux.Zeros() will switch bias off for the layer."

Should I modify 'conv_bn' function for this or just use the same conv_bn function here?

Reply:
Maybe add a `usebias = true` keyword argument for `conv_bn`? Then we can use it here with `usebias = false` and ignore everywhere else.